templates/zerver/help/change-your-org-name.md

```diff
@@ -0,0 +1,18 @@
+# Change your organization's name
+Only an administrator can change the name of the organization.
+
+Follow the following steps to change the name of the organization using an administrator account:
+* Click on the cog (![cog](/static/images/help/cog.png)) icon in the upper right corner
+of the right sidebar.
+
+* Choose **Administration** in the panel below:
+
+![admin-panel](/static/images/help/admin.png)
+
+* The following window will appear:
```

Comment:
Maybe add a brief description of the window? For example, you could add more details about the **Organization settings** section.